SoilMoistureSensor/Program.cs

```diff
@@ -1,17 +1,12 @@
-﻿using Microsoft.Azure.Devices.Client;
-
-Console.WriteLine();
-
-// var conn = Microsoft.Azure.Devices.Client.IotHubConnectionStringBuilder.Create("");
-// var client = Microsoft.Azure.Devices.Client.DeviceClient.Create(conn);
-
-// IOT connection
-// Don't need CounterFit - can just send any data
-
-
-// Connection to IOT
-// Create client for IOT & connect
-
-// Test project for soil moisture sensor
-// Test project for dashboard
-// Dashboard1 project
\ No newline at end of file
+﻿Console.WriteLine("Connecting");
+var IoTHubService = new IoTHubService("HostName=ECM3440.azure-devices.net;DeviceId=soil-moisture-sensor;SharedAccessKey=VGlo7rYmMZJVcVaJIehiOgF0QnUCmh3FHFxfpu9RukY=");
+Console.WriteLine("Connected");
+
+while (true)
+{
+    var rand = new Random();
+    var message = new SoilMoistureStatus { Value = rand.NextDouble() * 1023 };
```

Comment:
I'd make a ctor on SoilMoistureStatus so you don't have to do this property define